CMakeLists.txt

```
@@ -405,6 +405,11 @@ configure_file(
     ${PROJECT_BINARY_DIR}/${CMAKE_INSTALL_DATAROOTDIR}/modulefiles/${PROJECT_NAME}/${ROCPROFSYS_VERSION}
     @ONLY)
 
+configure_file(
+    ${PROJECT_SOURCE_DIR}/scripts/merge_multiprocess_output.sh
+    ${PROJECT_BINARY_DIR}/${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/bin/merge_multiprocess_output.sh
```

```suggestion
    ${PROJECT_SOURCE_DIR}/scripts/merge-multiprocess-output.sh
    ${PROJECT_BINARY_DIR}/${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/bin/merge-multiprocess-output.sh
```
Updating to account for renaming.